src/eShop.ServiceDefaults/OpenApi.Extensions.cs

```diff
@@ -21,45 +22,13 @@ public static IApplicationBuilder UseDefaultOpenApi(this WebApplication app)
             return app;
         }
 
-        app.UseSwagger();
+        app.MapOpenApi();
 
         if (app.Environment.IsDevelopment())
         {
-            app.UseSwaggerUI(setup =>
-            {
-                /// {
-                ///   "OpenApi": {
-                ///     "Endpoint: {
-                ///         "Name": 
-                ///     },
-                ///     "Auth": {
-                ///         "ClientId": ..,
-                ///         "AppName": ..
-                ///     }
-                ///   }
-                /// }
-
-                var pathBase = configuration["PATH_BASE"] ?? string.Empty;
-                var authSection = openApiSection.GetSection("Auth");
-                var endpointSection = openApiSection.GetRequiredSection("Endpoint");
-
-                foreach (var description in app.DescribeApiVersions())
-                {
-                    var name = description.GroupName;
-                    var url = endpointSection["Url"] ?? $"{pathBase}/swagger/{name}/swagger.json";
-
-                    setup.SwaggerEndpoint(url, name);
-                }
-
-                if (authSection.Exists())
-                {
-                    setup.OAuthClientId(authSection.GetRequiredValue("ClientId"));
-                    setup.OAuthAppName(authSection.GetRequiredValue("AppName"));
-                }
-            });
-
-            // Add a redirect from the root of the app to the swagger endpoint
-            app.MapGet("/", () => Results.Redirect("/swagger")).ExcludeFromDescription();
+            var authSection = openApiSection.GetSection("Auth");
```

Comment:
Remove?

Reply:
Yeah, we can remove this for now.

For context, Scalar hasn't shipped support for configuring OAuth through an options object so we can't actually use these configuration options yet.